common/core/web/keyboard-processor/src/text/keyboardProcessor.ts

```diff
@@ -448,25 +445,10 @@ namespace com.keyman.text {
           keyShiftState &= ~Codes.modifierCodes['RALT'];
         }
 
-        for(i=0; i < lockNames.length; i++) {
-          if(lockStates & Codes.stateBitmasks[lockNames[i]]) {
-            this.stateKeys[lockKeys[i]] = !!(lockStates & Codes.modifierCodes[lockNames[i]]);
-          }
-        }
-      } else if(d) {
-        keyShiftState |= v;
-
-        for(i=0; i < lockNames.length; i++) {
-          if(v & Codes.stateBitmasks[lockNames[i]]) {
-            this.stateKeys[lockKeys[i]] = true;
-          }
-        }
-      } else {
-        keyShiftState &= ~v;
-
-        for(i=0; i < lockNames.length; i++) {
-          if(v & Codes.stateBitmasks[lockNames[i]]) {
-            this.stateKeys[lockKeys[i]] = false;
```

Comment:
This was also a no-op. The only value passed in for `v` (at least when `e` was `null`) was `15` (`0x0F`), but `Codes.stateBitmasks` contains only values in the bitmask `0x3F00`.